src/Features/Onboarding.php

```diff
@@ -340,6 +340,7 @@ public static function get_allowed_product_types() {
 				'physical'      => array(
 					'label'       => __( 'Physical products', 'woocommerce-admin' ),
 					'description' => __( 'Products you ship to customers.', 'woocommerce-admin' ),
+					'preselect'   => true,
```

Comment:
What do you think about `default` here?  I'm thinking this would keep us slightly closer to semantics used in core - https://docs.woocommerce.com/document/settings-api/

That said, if you feel strongly that `preselect` is better we can keep it.

Reply:
Interesting idea Josh. Looking at the docs, seems like the `default` attribute could work nicely here.

Reply:
I'll go with `default` - naming things is hard ;-)

Reply:
Done in 4d802b3aed501d7cb4de4642cfeb1af2c8c53dae